src/System.Management.Automation/engine/CommandCompletion/ScopeArgumentCompleter.cs

```diff
@@ -29,16 +29,8 @@ public IEnumerable<CompletionResult> CompleteArgument(
             string wordToComplete,
             CommandAst commandAst,
             IDictionary fakeBoundParameters)
-        {
-            var scopePattern = WildcardPattern.Get(wordToComplete + "*", WildcardOptions.IgnoreCase);
-
-            foreach (string scope in s_Scopes)
-            {
-                if (scopePattern.IsMatch(scope))
-                {
-                    yield return new CompletionResult(scope);
-                }
-            }
-        }
+                => CompletionCompleters.EnumerateQuotedAndUnquotedCompletionText(
+                    wordToComplete,
+                    possibleCompletionValues: s_Scopes);
```

Comment:
```suggestion
                    possibleCompletionValues: s_Scopes,
                    resultType: CompletionResultType.ParameterValue);
```

Reply:
Good catch, fixed in b0a9dff8914cb2421f39a1366eea2eaae052fd78